src/components/views/elements/TintableSvgButton.js

```diff
@@ -39,9 +40,11 @@ export default class TintableSvgButton extends React.Component {
                     width={this.props.width}
                     height={this.props.height}
                 ></TintableSvg>
-                <span
+                <AccessibleButton
```

Comment:
Doesn't this AccessibleButton span end up containing nothing, so being zero width and sitting alongside the image?

Reply:
Nope, the (CSS) styling puts the span in front of the SVG with a width and height (12px x 12px). From memory, this is needed in order to prevent misleading pointer events from components of the SVG (e.g. path elements).